build-lab.sh

```diff
@@ -27,6 +27,10 @@ sudo apt-get install -y tree
 
 echo -e "${LINE_ASCII_CONSOLE}\n"
 
+echo "gem: --no-ri --no-rdoc" >> ${lab}/.gemrc
```

Comment:
Question: why do we no longer want documentation?

Reply:
If `.gemrc` does not already exist, which I do not believe it does, then you want to use a single `>` or else every time the student runs `build-lab.sh` a duplicate line will be appended to the `.gemrc` file.  The documentation provides the instruction, "If your containers are not running, then just run: `build-lab.sh`" in the [__Codespaces__](https://github.com/mitre/saf-training-lab-environment#codespaces) section, so the likelihood of a student executing the script multiple times is high.

Reply:
I discovered that the above `.gemrc` line won't work for two reasons:

1. The file has to be in the user's home directory, not the working directory.
2. The version of `gem` on Codespaces doesn't support those arguments; it uses the `--no-document` argument instead.

I'll create a new pull request with the fixes.